lib/src/index.rs

```diff
@@ -85,18 +110,26 @@ pub trait Index: Send + Sync {
         &self,
     ) -> Result<Box<dyn Iterator<Item = CommitId> + '_>, AllHeadsForGcUnsupported>;
 
+    /// Returns the subset of commit IDs in `candidates` which are heads in the
+    /// repository. If a commit id is duplicated in the `candidates` list it
+    /// will appear at most once in the output.
```

Comment:
"which are heads in the repository" is not quite correct. The function returns the candidates that are not ancestors of other candidates. So `heads(<root commit>)` will return the root commit even if there are other visible heads in the repo.

Reply:
Ok got it, thanks. I updated the comment.